package.json

```diff
@@ -11,7 +11,7 @@
   "license": "Apache-2.0",
   "main": "signalfx-lambda",
   "dependencies": {
-    "signalfx": "5.0.0"
+    "signalfx": "^7.0.0"
```

Comment:
Minor concern that we are using `^` now but weren't before. For libraries using `^` makes sense to me but was there a reason we didn't do it before? //cc @jwy @jaysfx

Reply:
@owais Not that I recall. The change seems fine to me though @jaysfx would know more about the effects of going from 5.0.0 to 7.0.0.

Reply:
Sorry, I haven't a clue.

Reply:
NodeJs runtimes in AWS are 10 & 12, so this seems OK to me.